src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -914,6 +934,8 @@ static void uct_rc_mlx5_ep_clean_qp(uct_rc_mlx5_ep_t *ep, struct ibv_qp *qp)
     uct_rc_mlx5_iface_common_update_cqs_ci(iface, &iface->super.super);
     (void)uct_ib_modify_qp(qp, IBV_QPS_RESET);
     uct_rc_mlx5_iface_common_sync_cqs_ci(iface, &iface->super.super);
+
+    uct_ib_iface_destroy_qp(qp);
```

Comment:
let's keep uct_ib_iface_destroy_qp(qp) outside this function to avoid mis-understanding